### PR TITLE
[HTD-48] Feat: Add `Term & Privacy content` Section

### PR DESCRIPTION
**Pull request by RBoraurito**
## Assets

| Light | Dark |
| ------ | ----- |
|    image    |    image   |

## Before Merge Checklist

The following steps were checked before merging a PR

- [x] checked and fixed code style issues
- [x] checked and removed unnecessary/junky code
- [x] manually tested introduced solution
- [ ] added needed tests to automate testing this code in the future

### CODE REVIEW

File: src/components/sections/rich-text/themes/terms-of-services/index.tsx
Comment (nephlin7) [inline src/components/sections/rich-text/themes/terms-of-services/index.tsx:1]: Can we rename this theme & code related to more generic names? Those styles/components will be the same for content pages, like terms of services & privacy policy, etc.

File: src/app/term-of-services/page.tsx [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+export default function TermOfServices() {
   return (
     <>
       <HeroPoints
         title="Terms Of Service"
         text={`**There's some legal text below which formally defines how you're allowed to use HTTP Toolkit, this website, and your HTTP Toolkit user account. It looks serious, but it's all fairly standard boilerplate.** \n\n **The rules here apply if you're using the hosted services (e.g. httptoolkit.com and app.httptoolkit.tech) and user accounts, or the published application binaries. This does not limit your open-source freedoms, and if you're building HTTP Toolkit yourself on your own infrastructure then only the open-source licenses apply.** \n\n **Note that the binaries specifically are dual-licensed: they're available under either the AGPL v3, guaranteeing open-source freedoms within the constraints of AGPL, or under the Creative Commons BY-ND license, allowing unencumbered licensing for simple unmodified use.**`}
       />
+      <ContentWithTable
+        content={Content}
+        links={[
```
Comment (nephlin7) [inline src/app/term-of-services/page.tsx:15]: Just to let you know, those links should be generated automatically based on the content headings. Let's leave it as it is in this PR, We will explore this [remark plugin](https://github.com/remarkjs/remark-toc) while working on the docs pages

Comment (RBoraurito) [reply src/app/term-of-services/page.tsx:15]: @nephlin7 I'll leave the comment

File: src/components/sections/rich-text/themes/terms-of-services/index.tsx
Comment (RBoraurito) [reply src/components/sections/rich-text/themes/terms-of-services/index.tsx:1]: I added a default theme that you could use to extend custom themes; you can move from the policies theme to default if needed.

Comment (nephlin7) [reply src/components/sections/rich-text/themes/terms-of-services/index.tsx:1]: @RBoraurito, but Do you think the h2,h3, spacer & ul will be different for other pages? I mean, Do you think it is necessary to create a theme for each content page? From what I can observe, the design of all the `richText` styles is identical across all pages where it is used.

**Review by nephlin7 — COMMENTED** [review]
https://github.com/bejamas/httptoolkit-website/pull/75#discussion_r1521520724

File: src/components/sections/rich-text/themes/terms-of-services/index.tsx
Comment (RBoraurito) [reply src/components/sections/rich-text/themes/terms-of-services/index.tsx:1]: I you want, I can take that on the rich text task, try to standardize default styles, what do you think?

Comment (nephlin7) [reply src/components/sections/rich-text/themes/terms-of-services/index.tsx:1]: Got it, leave a TODO message, please.

**Review by nephlin7 — APPROVED** [review]

**Pull request merged**